plugins/network-elements/nsx/src/main/java/org/apache/cloudstack/service/NsxElement.java

```diff
@@ -645,4 +663,76 @@ private List<NsxLoadBalancerMember> getLoadBalancerMembers(LoadBalancingRule lbR
         }
         return lbMembers;
     }
+
+    @Override
+    public boolean applyNetworkACLs(Network network, List<? extends NetworkACLItem> rules) throws ResourceUnavailableException {
```

Comment:
I think these methods are already merged on the nsx-integration branch, maybe we need to rebase?